src/app/pcpolicy/pcpolicy.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardService } from '../services/dashboard.service';
+
+@Component({
+  selector: 'app-pcpolicy',
+  templateUrl: './pcpolicy.component.html',
+  styleUrls: ['./pcpolicy.component.scss']
+})
```

Comment:
No, it is not required. when we generate a new component it is generated by default and has no semicolon.